config/test.exs

```diff
@@ -33,6 +33,8 @@ config :argon2_elixir, t_cost: 1, m_cost: 8
 # Disable Oban Queues
 config :keila, Oban, queues: false, plugins: false
 
+config :keila, Keila.Mailings, campaign_schedule_offset: -10
```

Comment:
Why do we configure a negative offset here? I think this would not be allowed in real usage so if we do it here to facilitate a single test or something we should at least put a comment here.

Ideally we would only configure this for the one test but I guess since it's a global config it's a bit harder (unless we would built in escape hatches that allow us to pass the param as an optional config but I'm not a hughe fan of that either).

Reply:
I’d say that the offset is mostly a matter of convenience and avoiding user confusion rather than ensuring system integrity. In the same vein, I think wanting to schedule tests at `utc_now` is very reasonable in the testing environment; having an offset of `0` might make tests fail occasionally - hence the negative value.

Reply:
Good points. What do you think about adding a comment here explaining this?

Reply:
Good idea, comment added!